source/languages/en/riak/dev/taste-of-riak/csharp.md

```diff
@@ -8,31 +8,28 @@ audience: beginner
 keywords: [developers, client, csharp]
 ---
 
-If you haven't set up a Riak Node and started it, please visit the [[Prerequisites|Taste of Riak: Prerequisites]] first.
+If you haven't set up a Riak Node and started it, please visit the
```

Comment:
It looks like these couple lines were changed so they would wrap at ~80-columns, but the rest of this file doesn't look like it wraps.
If the rest of the file doesn't wrap, we probably don't need to worry about these lines wrapping.